mooncake-common/etcd/etcd_wrapper.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+/*
+#include <stdint.h> 
+*/
+import "C"
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"unsafe"
+)
+
+type CClient struct {
+	client *clientv3.Client
+}
+
+//export NewEtcdClient
+func NewEtcdClient(endpoints *C.char) C.uintptr_t {
+	endpoint := C.GoString(endpoints)
+	cli, err := clientv3.New(clientv3.Config{
+		Endpoints:   []string{endpoint},
+		DialTimeout: 5 * time.Second,
+	})
+	if err != nil {
+		panic(fmt.Sprintf("failed to connect to etcd: %v", err))
+	}
+	client := &CClient{client: cli}
```

Comment:
we need to save the client into a model level global variable? otherwise it will be freed by GC.